ops/services/okta-global/main.tf

```diff
@@ -15,25 +15,31 @@ resource "okta_auth_server_scope" "sr" {
   default          = false
 }
 
-// Add the required claims
-resource "okta_auth_server_claim" "sr_user" {
-  auth_server_id = data.okta_auth_server.default.id
-  claim_type     = "RESOURCE"
-  name           = "org"
-  value          = "appuser.pdi_org"
-  scopes = [
-  okta_auth_server_scope.sr.name]
+resource "okta_auth_server_scope" "sr_prod" {
+  auth_server_id   = data.okta_auth_server.default.id
+  name             = "simple_report_prod"
+  description      = "Production-only OAUTH scope for Simple Report application"
+  metadata_publish = "NO_CLIENTS"
+  default          = false
 }
 
-resource "okta_auth_server_claim" "sr_org" {
-  auth_server_id = data.okta_auth_server.default.id
-  claim_type     = "RESOURCE"
-  name           = "access"
-  value          = "appuser.pdi_user"
-  scopes = [
-  okta_auth_server_scope.sr.name]
+resource "okta_auth_server_scope" "sr_stg" {
```

Comment:
This should be moved into the `Okta-app` module as well. Using the variable pattern described earlier.

```
resource "okta_auth_server_scope" "sr" {
   auth_server_id   = data.okta_auth_server.default.id
   name             = "simple_report_${var.env}"
   description      = "${var.env}-only OAUTH scope for Simple Report application"
   metadata_publish = "NO_CLIENTS"
   default          = false
 }
```